packages/shared/src/components/layout/common.tsx

```diff
@@ -40,14 +40,16 @@ export const algorithms = [
   { value: RankingAlgorithm.Popularity, text: 'Recommended' },
   { value: RankingAlgorithm.Time, text: 'By date' },
 ];
-const algorithmsList = algorithms.map((algo) => algo.text);
+export const algorithmsList = algorithms.map((algo) => algo.text);
 export const periods = [
   { value: 7, text: 'Last week' },
   { value: 30, text: 'Last month' },
   { value: 365, text: 'Last year' },
 ];
 const periodTexts = periods.map((period) => period.text);
 
+export const DEFAULT_ALGORITHM_KEY = 'feed:algorithm';
+
 export const SearchControlHeader = ({
```

Comment:
Why isn't this component in it's own file? seems like a decent one to have it's own file..

Reply:
Legacy 😂